app/controllers/v2/ManageKeywordsController.scala

```diff
@@ -84,27 +84,77 @@ class ManageKeywordsController @Inject()(
 
   def createKeyword(activeSubNav: SubNavigationTab = ManagerToolsKeywordsTab): Action[AnyContent] =
     (verify.authenticated andThen verify.mustHave(Permission.MANAGE_USERS)).async { implicit request =>
-
-      keywordService.findAll(NoPagination()).flatMap {
-        keywords =>
-          val keywordNames = keywords.results.map(_.name)
-          KeywordForm.formWithAuto(keywordNames).bindFromRequest.fold(
+      keywordService.findAll(NoPagination()).flatMap { keywords =>
```

Comment:
I think you might need to pull from master and update this as this method doesn't take a `Pagination` argument anymore